src/app/util/af-types.h

```diff
@@ -41,6 +41,9 @@
 
 #include <app-common/zap-generated/cluster-enums.h>
 #include <app-common/zap-generated/cluster-objects.h>
+#include <protocols/interaction_model/StatusCode.h>
+
+using chip::Protocols::InteractionModel::Status;
```

Comment:
`using` statements at global scope in headers are not OK.

Reply:
fixed in PR #32055